[2023-11-24T08:56:27.854+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projet1.install_matplotlib manual__2023-11-24T08:56:20.546912+00:00 [queued]>
[2023-11-24T08:56:27.864+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projet1.install_matplotlib manual__2023-11-24T08:56:20.546912+00:00 [queued]>
[2023-11-24T08:56:27.865+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-24T08:56:27.880+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_matplotlib> on 2023-11-24 08:56:20.546912+00:00
[2023-11-24T08:56:27.891+0000] {standard_task_runner.py:57} INFO - Started process 89 to run task
[2023-11-24T08:56:27.895+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Projet1', 'install_matplotlib', 'manual__2023-11-24T08:56:20.546912+00:00', '--job-id', '815', '--raw', '--subdir', 'DAGS_FOLDER/Projet.py', '--cfg-path', '/tmp/tmpyxo5ekg3']
[2023-11-24T08:56:27.896+0000] {standard_task_runner.py:85} INFO - Job 815: Subtask install_matplotlib
[2023-11-24T08:56:27.950+0000] {task_command.py:416} INFO - Running <TaskInstance: Projet1.install_matplotlib manual__2023-11-24T08:56:20.546912+00:00 [running]> on host b36ebcee5948
[2023-11-24T08:56:28.037+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='filali.dia18@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projet1' AIRFLOW_CTX_TASK_ID='install_matplotlib' AIRFLOW_CTX_EXECUTION_DATE='2023-11-24T08:56:20.546912+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-24T08:56:20.546912+00:00'
[2023-11-24T08:56:28.041+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-24T08:56:28.042+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install matplotlib']
[2023-11-24T08:56:28.059+0000] {subprocess.py:86} INFO - Output:
[2023-11-24T08:56:29.830+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-11-24T08:56:30.330+0000] {subprocess.py:93} INFO - Requirement already satisfied: matplotlib in /home/***/.local/lib/python3.8/site-packages (3.7.4)
[2023-11-24T08:56:30.343+0000] {subprocess.py:93} INFO - Requirement already satisfied: numpy<2,>=1.20 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (1.24.4)
[2023-11-24T08:56:30.343+0000] {subprocess.py:93} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (0.12.1)
[2023-11-24T08:56:30.345+0000] {subprocess.py:93} INFO - Requirement already satisfied: importlib-resources>=3.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (6.1.0)
[2023-11-24T08:56:30.346+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging>=20.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (23.2)
[2023-11-24T08:56:30.347+0000] {subprocess.py:93} INFO - Requirement already satisfied: python-dateutil>=2.7 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (2.8.2)
[2023-11-24T08:56:30.348+0000] {subprocess.py:93} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (10.1.0)
[2023-11-24T08:56:30.349+0000] {subprocess.py:93} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (4.45.1)
[2023-11-24T08:56:30.350+0000] {subprocess.py:93} INFO - Requirement already satisfied: contourpy>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (1.1.1)
[2023-11-24T08:56:30.350+0000] {subprocess.py:93} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (3.1.1)
[2023-11-24T08:56:30.351+0000] {subprocess.py:93} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib) (1.4.5)
[2023-11-24T08:56:30.449+0000] {subprocess.py:93} INFO - Requirement already satisfied: zipp>=3.1.0 in /home/***/.local/lib/python3.8/site-packages (from importlib-resources>=3.2.0->matplotlib) (3.17.0)
[2023-11-24T08:56:30.486+0000] {subprocess.py:93} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.8/site-packages (from python-dateutil>=2.7->matplotlib) (1.16.0)
[2023-11-24T08:56:37.745+0000] {subprocess.py:93} INFO - 
[2023-11-24T08:56:37.746+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.3.1
[2023-11-24T08:56:37.747+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-11-24T08:56:37.956+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-24T08:56:37.990+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Projet1, task_id=install_matplotlib, execution_date=20231124T085620, start_date=20231124T085627, end_date=20231124T085637
[2023-11-24T08:56:38.010+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-24T08:56:38.037+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
